src/pages/SessionPage/SessionPage.tsx

```diff
@@ -0,0 +1,10 @@
+import DefaultPageLayout from '@/components/DefaultPageLayout/DefaultPageLayout.tsx';
+import SessionPageContent from '@/components/SessionPageCpmponents/SessionPageContent';
+
+export default function SessionPage() {
+  return (
+    <DefaultPageLayout activeTab="groups">
```

Comment:
Тут надо отключить подсветку:
```suggestion
    <DefaultPageLayout>
```

Reply:
Иногда копипаста даёт сбои :с